skyportal/handlers/api/observation_plan.py

```diff
@@ -1575,3 +1577,191 @@ async def get(self, observation_plan_request_id):
         data = io.BytesIO(rez["data"])
 
         await self.send_file(data, filename, output_type=output_format)
+
+
+class DefaultObservationPlanRequestHandler(BaseHandler):
+    @auth_or_token
+    def post(self):
+        """
+        ---
+        description: Create default observation plan requests.
+        tags:
+          - default_observation_plan
+        requestBody:
+          content:
+            application/json:
+              schema: DefaultObservationPlanPost
+        responses:
+          200:
+            content:
+              application/json:
+                schema:
+                  allOf:
+                    - $ref: '#/components/schemas/Success'
+                    - type: object
+                      properties:
+                        data:
+                          type: object
+                          properties:
+                            id:
+                              type: integer
+                              description: New default observation plan request ID
+        """
+        data = self.get_json()
+
+        target_groups = []
+        for group_id in data.pop('target_group_ids', []):
+            g = Group.get_if_accessible_by(
+                group_id, self.current_user, raise_if_none=False
+            )
+            if g is None:
+                raise AttributeError(f"Missing group with ID: {group_id}")
+            target_groups.append(g)
+
+        allocation = Allocation.get_if_accessible_by(
+            data['allocation_id'],
+            self.current_user,
+            raise_if_none=False,
+        )
+        if allocation is None:
+            raise AttributeError(f"Missing allocation with ID: {data['allocation_id']}")
+
+        instrument = allocation.instrument
+        if instrument.api_classname_obsplan is None:
+            raise AttributeError('Instrument has no remote API.')
+
+        try:
```

Comment:
Is this just copied from the other handler, or does it still have a purpose?

Reply:
Yeah we need to check for the observation plan handler because not all of the instruments have them by default unfortunately.